src/components/AccountContainer.js

```diff
@@ -1,61 +1,37 @@
 import React, { Component } from 'react'
 import TransactionsList from './TransactionsList'
 import Search from './Search'
+const url = 'https://boiling-brook-94902.herokuapp.com/transactions'
```

Comment:
you might consider moving this into an adapter or separate function so that you don't have to define this here.